bms/bms_app/static/bms_app/css/create_prodStyle.css

```diff
@@ -0,0 +1,28 @@
+
+.createprod
+{
+    padding-left: 100px;
+}
+#navbrand
+{
+    padding-left: 200px;
+}
+#cancelbuton
```

Comment:
Two words should be seperated by  "-" eg. cancel-button.
Button spelling is wrong